web/src/main/java/com/futurewei/alcor/web/entity/subnet/SubnetEntity.java

```diff
@@ -28,7 +28,7 @@
 @Data
 public class SubnetEntity extends CustomerResource {
 
-    @JsonProperty("network_id")
+    @JsonProperty("vpc_id")
```

Comment:
Please revert this change. Externally we use "network_id" to be OpenStack compatible.

Reply:
changed in dcede47